.vib/keycloak/cypress/cypress/integration/keycloak_spec.js

```diff
@@ -25,9 +25,9 @@ it('import and check user information', () => {
       obj.users[0].username = `${user.importedUser.username}.${random}`;
       obj.users[0].email = `${random}.${user.importedUser.email}`;
       obj.users[0].lastName = `${user.importedUser.lastName}-${random}`;
-      const jsonText = JSON.stringify(obj);
-      cy.get('.view-line').type(jsonText, { parseSpecialCharSequences: false, delay: 1 });
+      cy.writeFile(importFile, JSON.stringify(obj));
```

Comment:
It is only a minor thing but we don't need to use `JSON.stringify()` here. The `obj` object that `readFile()` returns can be saved as it is, converting it to text is redundant.

Before this change we needed to transform `obj` into a `string` so it could be used by the `type()` function, but not anymore.

```suggestion
      cy.writeFile(importFile, obj);
```